types/validator_set.go

```diff
@@ -53,7 +53,7 @@ func (valSet *ValidatorSet) IncrementAccum(times int) {
 	// Add VotingPower * times to each validator and order into heap.
 	validatorsHeap := cmn.NewHeap()
 	for _, val := range valSet.Validators {
-		val.Accum += int64(val.VotingPower) * int64(times) // TODO: mind overflow
+		val.Accum += val.VotingPower * int64(times) // TODO: mind overflow
```

Comment:
this one's actually a little scary.

Reply:
very scary

Reply:
opened an issue https://github.com/tendermint/tendermint/issues/919